5.process_CP_features/scripts/2.fuzzy_matching.py

```diff
@@ -107,13 +107,20 @@
 # In[5]:
 
 
+tracked_cells_stats = {
+    "Metadata_time": [],
+    "total_CP_cells": [],
+    "total_annotated_cells": [],
+}
 for time in profiles["Metadata_Time"].unique():
     df_left = profiles.copy().loc[profiles["Metadata_Time"] == time]
     df_right = tracks.copy().loc[tracks["Metadata_t"] == time]
 
     total_CP_cells += df_left.shape[0]
+
     # loop through the rows in the subset_annotated_df and find the closest coordinate set in the location metadata
     for index1, row1 in df_left.iterrows():
+        tracked_cells_stats["total_CP_cells"].append(1)
```

Comment:
I am a bit confused by this section:

You append 1 to the total CP cell count, but then I don't see it being updated, so it looks like every loop will assign the total cells to 1. Or maybe I am misunderstanding this section. I would assume you would need to have another piece of code to add the number of rows as the total CP cells or something. 

Recommend including some more code comments in here about what this is doing.